src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1792,6 +1821,150 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly mutationEnrichmentProfiles = remoteData<MolecularProfile[]>({
+        await: () => [
+            this.molecularProfilesInStudies
+        ],
+        invoke: async () => {
+            const profiles: MolecularProfile[] = _.filter(this.molecularProfilesInStudies.result, 
+                (profile: MolecularProfile) => profile.molecularAlterationType === AlterationTypeConstants.MUTATION_EXTENDED);
+            this.selectedEnrichmentMutationProfile = profiles[0];
+            return profiles;
+        }
+    });
+
+    readonly mutationEnrichmentData = remoteData<AlterationEnrichment[]>({
+        await: () => [
+            this.alteredSamples,
+            this.unalteredSamples,
+            this.mutationEnrichmentProfiles
+        ],
+        invoke: async () => {
+            return this.selectedEnrichmentMutationProfile ? this.sortEnrichmentData(
```

Comment:
maybe a comment here.  seems like this would return either a single profile or a an array of profiles?